### ISSUE TITLE
Still alive?

### ISSUE THREAD

**Issue by HofmannZ** [thebuilder/react-scroll-percentage#175]
Is this project still alive, can't really find any good alternative for it?

**Comment by thebuilder** [comment]
@HofmannZ not really. It should still work, but I haven't had the need to use it myself - I'd recommend using a maintained animation library like Framer Motion instead, which can do the same in performant way.

**Comment by IanBoyte** [comment]
@thebuilder Why would you recommend a package that's 10 times the gzipped size of the package that does exactly what you need?
